@navikt/core/react/src/util/composeEventHandlers.ts

```diff
@@ -0,0 +1,22 @@
+/* https://github.com/radix-ui/primitives/blob/main/packages/core/primitive/src/primitive.tsx#L1 */
+
+/**
+ * Utility to consistently call original eventhandler, often from props and internal eventhandler
+ * @internal
+ */
+export function composeEventHandlers<T>(
```

Comment:
Ser ut som dette kan gjøre at vi slipper `as unknown as Event` på linje 17

```suggestion
export function composeEventHandlers<T extends React.SyntheticEvent>(
```